packages/module/patternfly-docs/content/examples/FilterSidePanel.md

```diff
@@ -118,7 +118,7 @@ class MockFilterSidePanelExample extends React.Component {
     const maxShowCount = 5;
     const leeway = 2;
     return (
-     <div style={{ width: '205px', border: '1px solid grey', paddingTop: '20px' }}>   
+     <div style={{ width: '205px' }}>   
```

Comment:
Removed the border and example padding so it's clear what's coming from `<FilterSidePanel>` and what's part of the example. Left the width because there is no width on `<FilterSidePanel>` and I'm assuming this panel goes into some parent/container with a constrained width, so this will mimic closer how the component will display. Ideally I'd probably drop this width, too, so you're just seeing the extension styles.

Just dropping a note here but this and the properties side panel seem to share some styling - they're narrow, probably have padding around them. And `<FilterSidePanel>` has padding and no width, and `<PropertiesSidePanel>` has a width and no padding. I'm guessing that's fine, but wondering if there could be some consistency there.